src/libraries/System.Text.Json/Common/JsonSourceGenerationOptionsAttribute.cs

```diff
@@ -125,6 +125,16 @@ public JsonSourceGenerationOptionsAttribute(JsonSerializerDefaults defaults)
         /// </summary>
         public bool WriteIndented { get; set; }
 
+        /// <summary>
+        /// Specifies the default value of <see cref="JsonSerializerOptions.IndentCharacter"/> when set.
+        /// </summary>
+        public char IndentCharacter { get; set; } = JsonConstants.DefaultIndentCharacter;
```

Comment:
Following convention of other properties in this attribute, there's no need to specify a default value here since these values are only read at compile time.

Reply:
Removed.